packages/runtime/datastore/src/channelContext.ts

```diff
@@ -6,12 +6,18 @@
 import { IChannel } from "@fluidframework/datastore-definitions";
 import { IDocumentStorageService } from "@fluidframework/driver-definitions";
 import { ISequencedDocumentMessage, ISnapshotTree } from "@fluidframework/protocol-definitions";
-import { IChannelSummarizeResult, IContextSummarizeResult, IGCData } from "@fluidframework/runtime-definitions";
+import {
+    IChannelSummarizeResult,
+    IContextSummarizeResult,
+    IGCData,
+    IGCDetails,
+} from "@fluidframework/runtime-definitions";
 import { addBlobToSummary } from "@fluidframework/runtime-utils";
 import { ChannelDeltaConnection } from "./channelDeltaConnection";
 import { ChannelStorageService } from "./channelStorageService";
 
 export const attributesBlobKey = ".attributes";
+export const gcBlobKey = ".gc";
```

Comment:
Same here - we need to either reduce number of blobs, or remove dot from it to be less important in term of inclusion into snapshot (and not be on critical boot path)